src/main/java/it/pn/frontend/e2e/rest/RestNotification.java

```diff
@@ -37,15 +38,16 @@ public NewNotificationResponse newNotificationWithOneRecipientAndDocument(NewNot
         return null;
     }
 
-    public String getNotificationStatus(String notificationRequestId) {
+    public LinkedTreeMap<String,Object> getNotificationStatus(String notificationRequestId) {
         final CustomHttpClient<Object, Object> httpClient2 = CustomHttpClient.getInstance();  // Modifica qui
+        httpClient2.setBaseUrlApi("https://api.test.notifichedigitali.it");
         try {
             Object response = httpClient2.sendHttpGetRequest("/delivery/v2.3/requests?notificationRequestId=" + notificationRequestId, null, Object.class);
             if (response instanceof LinkedTreeMap) {
                 LinkedTreeMap<String, Object> responseData = (LinkedTreeMap<String, Object>) response;
                 if (responseData.containsKey("notificationRequestStatus")) {
-                    return responseData.get("notificationRequestStatus").toString();
-                } else {
+                    return responseData;
+                }else {
                     logger.error("L'attributo 'notificationRequestStatus' non è presente nella risposta JSON");
                     return null;
                 }
```

Comment:
It is not clear to me what's the attempt to convert it into a `LinkedTreeMap`

Reply:
so as to have the choice in the future to take different data and not just the request id